PSModule/InvokeAlmLabManagementTaskCmdlet.cs

```diff
@@ -8,54 +8,61 @@
 using System.Threading.Tasks;
 using PSModule.AlmLabMgmtClient.Result.Model;
 using PSModule.AlmLabMgmtClient.SDK;
-using PSModule.AlmLabMgmtClient.SDK.Interface;
 using PSModule.AlmLabMgmtClient.SDK.Util;
 
 namespace PSModule
 {
     using C = Constants;
-    using H = Helper;
 
     [Cmdlet(VerbsLifecycle.Invoke, "AlmLabManagementTask")]
     public class InvokeAlmLabManagementTaskCmdlet : AbstractLauncherTaskCmdlet
     {
         [Parameter(Position = 0, Mandatory = true)]
         public string ALMServerPath { get; set; }
 
-        [Parameter(Position = 1, Mandatory = true)]
-        public string ALMUserName { get; set; }
+        [Parameter(Position = 1, Mandatory = false)]
+        public bool IsSSO { get; set; }
 
         [Parameter(Position = 2)]
+        public string ClientID { get; set; }
+
+        [Parameter(Position = 3)]
+        public string ApiKeySecret { get; set; }
+
+        [Parameter(Position = 4, Mandatory = true)]
+        public string ALMUserName { get; set; }

```

Comment:
if `IsSSO` is `true`, is `ALMUserName` still mandatory?

Reply:
Good catch, it seems to work either way, but will change it in order to avoid any confusion.